package.json

```diff
@@ -23,11 +23,14 @@
   ],
   "license": "OGL-UK-3.0",
   "dependencies": {
+    "@apollo/server": "4.11.2",
+    "@as-integrations/hapi": "2.0.1",
     "@azure/identity": "4.4.1",
     "@hapi/hapi": "21.3.2",
     "applicationinsights": "2.9.1",
     "convict": "^6.2.4",
```

Comment:
As mentioned in a thread yesterday - remove the carets (`^`) from the package versions. There is a way to configure this on your local machine so that whenever you install packages it won't be included.

Reply:
Will also add I don't _think_ this is urgent but should be a quick fix?